.github/workflows/azwi-build.yaml

```diff
@@ -27,7 +27,7 @@ jobs:
           fetch-depth: 0
       - uses: actions/setup-go@v3
         with:
-          go-version: "^1.18"
+          go-version: "^1.19"
```

Comment:
I believe caret will upgrade to 1.19, would we want to pin these like `1.19`?

Reply:
Updated it to `1.19`